src/index.ts

```diff
@@ -226,8 +226,8 @@ export default class Common {
       hardforks = this.hardforks()
     }
 
-    let posHf1 = 0,
-      posHf2 = 0
+    let posHf1 = -1,
```

Comment:
Now non-active hardforks will not be considered equal to `chainstart` when `onlyActive` is passed. That makes sense.